packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { SvgIconProps } from '@material-ui/core/SvgIcon';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckIcon from '@material-ui/icons/Check';
+import { GridCellParams } from '../../models/params/gridCellParams';
+
+export function GridBooleanCell(props: GridCellParams & SvgIconProps) {
+  const {
+    id,
+    value,
+    element,
+    formattedValue,
+    api,
+    field,
+    row,
+    colDef,
+    cellMode,
+    getValue,
+    rowIndex,
+    colIndex,
+    isEditable,
+    ...other
+  } = props;
+
+  const Icon = value ? CheckIcon : CloseIcon;
+
+  return (
+    <Icon
+      fontSize="small"
+      className="MuiDataGrid-booleanCell"
+      aria-label={api.getLocaleText(value ? 'booleanCellTrueLabel' : 'booleanCellFalseLabel')}
+      data-value={Boolean(value)}
+      {...other}
```

Comment:
This is not working. VoiceOver reads as "Blank".

<img width="517" alt="Screenshot 2021-04-11 at 13 02 00" src="https://user-images.githubusercontent.com/3165635/114301656-2501a900-9ac6-11eb-8a39-2f42bb78a2f8.png">

Reply:
The first option is https://next.material-ui.com/components/icons/#semantic-icons

```diff
diff --git a/packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx b/packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx
index 742247f1..5c7a401c 100644
--- a/packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx
+++ b/packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx
@@ -28,7 +28,7 @@ export function GridBooleanCell(props: GridCellParams & SvgIconProps) {
     <Icon
       fontSize="small"
       className="MuiDataGrid-booleanCell"
-      aria-label={api.getLocaleText(value ? 'booleanCellTrueLabel' : 'booleanCellFalseLabel')}
+      titleAccess={api.getLocaleText(value ? 'booleanCellTrueLabel' : 'booleanCellFalseLabel')}
       data-value={Boolean(value)}
       {...other}
     />
```

Then, it will read the correct value "true" or "false".

Reply:
I also had a quick look into https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/checkbox/checkbox-1.html, removing the need for `booleanCellTrueLabel` i18n but I did look futher.